crates/rpc-types-beacon/src/requests.rs

```diff
@@ -0,0 +1,150 @@
+use alloy_eips::{
+    eip6110::DepositRequest, eip7002::WithdrawalRequest, eip7251::ConsolidationRequest,
+};
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(deny_unknown_fields)]
+#[serde(rename_all = "snake_case")]
+#[cfg_attr(feature = "ssz", derive(ssz_derive::Decode, ssz_derive::Encode))]
+/// An Electra-compatible execution requests payload.
+pub struct ExecutionRequestsV4 {
+    /// The requested deposits.
+    pub deposits: Vec<DepositRequest>,
+    /// The requested withdrawals.
+    pub withdrawals: Vec<WithdrawalRequest>,
+    /// The requested consolidations.
+    pub consolidations: Vec<ConsolidationRequest>,
+}
+
+#[cfg(feature = "ssz_types")]
+mod from_requests {
+    use crate::requests::ExecutionRequestsV4;
+    use alloy_eips::{
+        eip6110::DepositRequest, eip7002::WithdrawalRequest, eip7251::ConsolidationRequest,
+        eip7685::Requests,
+    };
+    use ssz::DecodeError;
+
+    impl TryFrom<&Requests> for ExecutionRequestsV4 {
+        type Error = Error;
+        fn try_from(value: &Requests) -> Result<Self, Self::Error> {
+            use alloy_eips::{
+                eip6110::DEPOSIT_REQUEST_TYPE, eip7002::WITHDRAWAL_REQUEST_TYPE,
+                eip7251::CONSOLIDATION_REQUEST_TYPE,
+            };
+            use ssz::Decode;
+            use ssz_types::{typenum, VariableList};
```

Comment:
let's move those imports above

Reply:
Yup!  Moved as part of the larger changes in https://github.com/alloy-rs/alloy/pull/1787/commits/cac4fab81059957caea3ff13e9236be4f8d6566c.